plugins/snowflake/dbt/adapters/snowflake/connections.py

```diff
@@ -245,13 +246,17 @@ def cancel(self, connection):
         logger.debug("Cancel query '{}': {}".format(connection_name, res))
 
     @classmethod
-    def get_status(cls, cursor):
+    def get_status(cls, cursor) -> ExecutionStatus:
         state = cursor.sqlstate
 
         if state is None:
             state = 'SUCCESS'
 
-        return "{} {}".format(state, cursor.rowcount)
+        return ExecutionStatus(
+            message="{} {}".format(state, cursor.rowcount),
+            rows=cursor.rowcount,
```

Comment:
Snowflake always reports back `SUCCESS 1` for `create view` + `create table` statements, even if the resulting view or table has many more rows. I don't think it's really our place to intervene here, though, since we're pulling this value right off the connection

Reply:
same as above..